torchgeo/datamodules/landcoverai.py

```diff
@@ -36,6 +38,43 @@ def __init__(
         self.batch_size = batch_size
         self.num_workers = num_workers
 
+    def on_after_batch_transfer(
+        self, batch: Dict[str, Any], batch_idx: int
+    ) -> Dict[str, Any]:
+        """Apply batch augmentations after batch is transferred to the device.
+
+        Args:
+            batch: mini-batch of data
+            batch_idx: batch index
+
+        Returns:
+            augmented mini-batch
+        """
+        try:
+            if self.trainer.training:  # type: ignore[union-attr]
+                # Kornia expects masks to be floats with a channel dimension
+                x = batch["image"]
+                y = batch["mask"].float().unsqueeze(1)
+
+                train_augmentations = K.AugmentationSequential(
```

Comment:
I think we should make the augs be an input arg to the datamodule. Users will want to use different augmentations. E.g. I don't ever use RandomSharpness (I think RandomBlur is a better option) so I would already have to manually remove this from the datamodule. This can come in a different PR though.

Reply:
Sounds interesting, but let's save it for a future PR.